source/_integrations/owntracks.markdown

```diff
@@ -3,6 +3,7 @@ title: OwnTracks
 description: Instructions on how to use Owntracks to track devices in Home Assistant.
 ha_category:
   - Presence Detection
+ha_iot_class: Cloud Polling
```

Comment:
This is going to sound weird, but this is actually local push.

There is no third-party cloud involved, hence it should be local. Furthermore, the clients push updates directly into your Home Assistant instance.

```suggestion
ha_iot_class: Local Push
```

Reply:
This was a difficult one, thanks!